.storybook/manager-head.html

```diff
@@ -14,7 +14,7 @@
         format('woff2'),
       url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold.ttf')
         format('truetype');
-    font-weight: 700, bold;
+    font-weight: 700;
```

Comment:
Removing bold since major browsers map `bold` to 700

Reply:
Why is this change necessary? I'm approving the change, but I feel like we should all know why this is being done.

Should this change also be applied to the website? Looks like we've got `700, bold` defined there too.

Reply:
Safari and Firefox don't support this way of defining font-face rules. I considered duplicating these definitions with `700` and `bold`, but it seems we don't have to. 
Good callout on the website, I'll fix that

Reply:
Makes sense! Thanks for clarifying.